backend/log.go

```diff
@@ -0,0 +1,8 @@
+package backend
+
+import (
+	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
+)
+
+// Logger is the default logger instance.
+var Logger log.Logger = log.New()
```

Comment:
Do we really need a global variable? People should new this up in their plugin IMO.

Not a blocker. just a matter of opinions.

Reply:
or maybe figure out a way to inject it?

Reply:
In general I don't like global variables for the standard reasons. However, I found when working with backend plugins that I want to log all over place (at least for debugging).

This means that if my plugin has multiple packages, I need to figure out how to initialaize logging in each package, or may pass it through somehow.

Therefore in this case I like the global variables because plugin authors will be able to log easily without having to think about it.

Reply:
Currently there's no support in Grafana for naming a plugin logger why making a global one available is of convenience. 

Thinking about this we may want to create a specific named logger in Grafana that captures everything in the backend plugin - with this we would allow log levels to be configured per backend plugin in Grafana instead of current where you can change log level for everything backend plugin related and not per plugin.

Reply:
Will try to get this merged, but will update the issue and keep it open for further improvements.